test/backend/inference_backend.py

```diff
@@ -715,7 +718,7 @@ def get_test_models():
         "test_reduce_l2_negative_axes_keep_dims_example_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{-1}},
         "test_reduce_l2_negative_axes_keep_dims_random_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{-1}},
 
-        # ==OP== #ReduceMax
+        # ==OP== ReduceMax
```

Comment:
This caused `make onnx_mlir_supported_ops` failed to generate doccument files for supported ops. Fix it here.